pkg/sql/colexec/agg/count/count.go

```diff
@@ -145,3 +147,222 @@ func (r *Count) Eval(zs []int64, _ *mheap.Mheap) *vector.Vector {
 		Nsp:  &nulls.Nulls{},
 	}
 }
+
+//for distinct count, the arguments can be many, So I make a simple way to splve it (need to optimize)
+//like count(distinct a,b,c),I will parse a,b,c as string type to get stra,strb,strc. Then I will use
+//Crc32 to hash them and I get uint32 of a,b,c, they are u32a,u32b,u32c. then add them all u32a+u32b+u32c
+//so I get the hashvalue of multicols. But I don't know whether its effect is good.(todo: optimize it)
```

Comment:
Wait -- are you using Crc32 to count distinct?    This won't work.

Reply:
Yes, this may conflict

Reply:
Yeah, I have modified. Before, I think of the count(distinct a,b,c), and I found a way: that's up. But now I know it's bad. Now, we just think about the only one columns. If we think about multi columns. I will refer to the group.go

Reply:
Even one column won't work.   CRC is simply not good enough.    Why not just count distinct tuples?

Reply:
I have talk him to refer to the logic of join, make a key out of each column and insert it into the hashtable.